treenode.go

```diff
@@ -659,14 +659,14 @@ func (n *TreeNodeInstance) SendToChildrenInParallel(msg interface{}) error {
 		go func(n2 *TreeNode) {
 			if err := n.SendTo(n2, msg); err != nil {
 				eMut.Lock()
-				errs = append(errs, collectedErrors{name, err})
+				errs = append(errs, errors.New(name+": "+err.Error()))
```

Comment:
You don't need to do this right now, but it is better to keep error info in machine readable format, instead of collapsing it into strings. So instead of errors.New, you'd make a new struct called NodeError{ Node *TreeNode, Error error } and append those onto errs. Type NodeError would need to implement method "Error() string" in order to implement error.